pdc_client/runner.py

```diff
@@ -72,8 +72,13 @@ def load_plugins(self):
             raise TypeError('Server must be specified')
         config = pdc_client.read_config_file(server)
         plugins = DEFAULT_PLUGINS
-        if config and config[CONFIG_PLUGINS_KEY_NAME]:
-            plugins = config[CONFIG_PLUGINS_KEY_NAME]
+        if config:
+            try:
+                plugins = config[CONFIG_PLUGINS_KEY_NAME]
+            except KeyError:
+                pass
+        if not isinstance(plugins, list):
+            raise TypeError('Plugins must be a list')
```

Comment:
for the json file, it can be a dict, but the setting is "plugins": ["image.py", "rpm.py"...], i think list is the proper format. and for set, it is a python type, i don't know if there is same idea in json.